components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -29,18 +29,20 @@ class TextFieldTests: XCTestCase {
     XCTAssertEqual(textField.attributedText?.string, string)
   }
 
-  func testCopyingTextField() {
+  func testCopying() {
     let textField = MDCTextField()
 
     textField.clearButtonColor = .red
```

Comment:
Yes, so should we confirm that we are explicitly setting it to .none in the commonInit?